modules/parquet/test/index.js

```diff
@@ -1,17 +1,19 @@
-import './init';
+// import './init';
 
-// parquetjs test suite
-import './parquetjs/codec-plain.spec';
-import './parquetjs/codec-rle.spec';
-import './parquetjs/schema.spec';
-import './parquetjs/shred.spec';
-import './parquetjs/thrift.spec';
-// import './parquetjs/reader.spec';
+// // parquetjs test suite
```

Comment:
I assume this is temporary?

Reply:
Yes of course. Just to quickly test only the new test cases